sites.json

```diff
@@ -2667,6 +2667,18 @@
     },
 
     {
+        "name": "Hackerrank",
+        "url": "https://www.hackerrank.com/settings/account",
+        "difficulty": "easy",
+        "notes": "According to the site you must go to the profile settings url and there will be a button called 'delete account' which will delete the account if it is wished to. ",
+        "domains": [
+            "www.hackerrank.com"
+        ]
+    },
+
+
```

Comment:
Can you please condense these three newlines separating "Hackerrank" and "Hackpad" into one newline? Thanks!